hca/accession_mapper.py

```diff
@@ -0,0 +1,7 @@
+class AccessionMapper:
+    mapping: [str]
+    type: str
+
+    def __init__(self, mapping: [str], accession_type: str):
+        self.mapping = mapping
+        self.type = accession_type
```

Comment:
what is this class for? Do you need this?

Reply:
@aaclan-ebi Yep. This is used for the various entities accession mapping and define if the accession(s) is string or in an array